### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by zizdepski**

### CODE REVIEW

File: server/api/user.js [hunk 1]
```diff
@@ -25,6 +25,20 @@ const login = (req, res) => {
   });
 };
 
+const deleteUser = (req, res) => {
+  console.log('we getting here?', 2929292929);
```
Comment (jstrzesz2) [inline server/api/user.js:29]: remove console.logs before merge

File: server/api/user.js [hunk 2]
```diff
@@ -25,6 +25,20 @@ const login = (req, res) => {
...
+  const { username } = req.params;
+
+  User.destroy({
+    where: {
+      username,
+    },
+  }).then(() => {
+    console.log(`User ${username} was deleted`);
```
Comment (jstrzesz2) [inline server/api/user.js:37]: remove console.log before merge

File: server/api/user.js [hunk 3]
```diff
@@ -25,6 +25,20 @@ const login = (req, res) => {
...
+  }).catch((err) => {
+    console.error(err);
```
Comment (jstrzesz2) [inline server/api/user.js:39]: convert to console.warn

File: server/api/user.js (hunk 2 above)
Comment (zizdepski) [reply server/api/user.js:37]: Will do. Missed that one, thanks.

File: server/api/user.js (hunk 3 above)
Comment (zizdepski) [reply server/api/user.js:39]: Is that the solution to the eslint warnings I get for console.error?

**Pull request merged**